converters/fromZigbee.js

```diff
@@ -805,6 +805,20 @@ const converters = {
             };
         },
     },
+    ias_smoke_alarm_1_develco: {
```

Comment:
Why is there a separate converter needed?

Reply:
Thanks, for some reason I cannot make changes to this file, can you move this converter below this line: https://github.com/Koenkk/zigbee-herdsman-converters/blob/13e64802762374a66dc9b53811ef2c376c4643bb/converters/fromZigbee.js#L1443

Reply:
@Koenkk, I'll create a new PR for everything related to the SMSZB-120. Give me 2 minutes